apps/catalog/catalog/src/app/marketplace/title/avails/map/avails-map.component.ts

```diff
@@ -151,6 +152,11 @@ export class MarketplaceMovieAvailsMapComponent implements AfterViewInit, OnDest
     const decodedData = decodeUrl<MapAvailsFilter>(this.route);
     this.load(decodedData);
 
+    if (isPast(decodedData.duration?.from) || isPast(decodedData.duration?.to)) {
```

Comment:
to avoid entering function with undefined value : 
```suggestion
    if (decodedData.duration && (isPast(decodedData.duration.from) || isPast(decodedData.duration.to))) {
```